lints/lint_ext_san_dns_name_too_long.go

```diff
@@ -0,0 +1,38 @@
+// lint_ext_san_dns_name_too_long.go
+
+package lints
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type SANDNSTooLong struct {
+	// Internal data here
+}
+
+func (l *SANDNSTooLong) Initialize() error {
+	return nil
+}
+
+func (l *SANDNSTooLong) CheckApplies(c *x509.Certificate) bool {
+	return util.IsExtInCert(c, util.SANOID)
```

Comment:
so Zakir is right, it's possible there are no DNSNames and only IPs in a SAN extension, does ZLint handle this gracefully?